plutus-use-cases/test/Spec/renderCrowdfunding.txt

```diff
@@ -318,102 +318,6 @@ Balances Carried Forward:
   Value:
     Ada:      Lovelace:  99999965
 
-  Script: 55aa9cddd1f504a892c55412984bbdcc854800af1ccd2ba7b5c188129a3fb9d8
+  Script: 717395feb16ec55d2230e397a08addded61a2742a82ffc7e8838813ea3bafa3d
   Value:
-    Ada:      Lovelace:  225
-
-==== Slot #2, Tx #0 ====
```

Comment:
Hmm, this seems like it actually has fewer transactions in it now! That seems strange.

Reply:
Solved!

Reply:
What was it, out of interest?

Reply:
In the file `plutus-use-cases/test/Spec/Rollup.hs`, I simply changed `takeUntilSlot 20` to `takeUntilSlot 21` because now the deadline time is inclusive.